docs/source/getting_started/running_at_scale.ipynb

```diff
@@ -0,0 +1,148 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running Algorithms at Scale\n",
+    "\n",
+    "In order to run algorithms in the scaled-up cloud compute environment, they must first be \"registered\" in the Algorithm Catalog. This will make them available to other MAAP users, clearly define their inputs and outputs, and prepare them to be run easily in the Data Processing System (DPS).\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Register an Algorithm\n",
+    "\n",
+    "### Clone a test algorithm\n",
+    "\n",
+    "**[SHOULD THIS BE IN THE WRITING CODE SECTION?]**\n",
```

Comment:
I like this here to keep all algorithm related stuff in the same section. I could see a user just coming to read the docs for how to run an algorithm and I know this kinda falls under showing how to use GitHub in the writing code section, but we don't want to be repeating how to do this.
Maybe a solution could be name the header to something like "See GitHub branches and changes" so the user can find it better if they are looking for this information?

Reply:
I feel like maybe "writing code" should have clone-a-repo in it. In terms of a Getting Started we should have a clear path through:
- get your account
- make a workspace
- clone a repo so you have some example code
- change the code ("Writing code") and push it back to github (fork?)
- register "your" new algorithm
- run the algorithm on DPS and watch it finish

I think I should also make a high-level overview of what the Getting Started tutorial will include.. maybe relate it to the overview diagram.

Reply:
will consider for future revision